### ISSUE TITLE
Help:  unhealthy 

### ISSUE THREAD

**Issue by MilorES** [qdm12/ddns-updater#287]
**TLDR**: *I use the v2 token as indicated... although it seems that it works the error is coming out. 
Sorry for the mistakes, I only speak the Spanish language ... 

https://github.com/qdm12/ddns-updater/discussions
*

1. Is this urgent: No
2. DNS provider(s) you use: freedns
3. Program version:

Running version latest built on 2021-12-15T09:57:37Z (commit ae7f104)

4. What are you using to run the container: docker-compose
5. Extra information (optional)

Logs:

```
2022/01/05 15:00:03 INFO reading JSON config from file /updater/data/config.json

2022/01/05 15:00:03 INFO Found 2 settings to update records

2022/01/05 15:00:04 INFO Reading history from database: domain mooo.com host ost-tecnomatic

2022/01/05 15:00:04 INFO Reading history from database: domain mooo.com host soporte-tecnomatic

2022/01/05 15:00:04 INFO healthcheck server: listening on 127.0.0.1:9999

2022/01/05 15:00:04 INFO backup: disabled

2022/01/05 15:00:04 INFO http server: listening on :8000

2022/01/05 15:00:04 INFO IPv4 address of ost-tecnomatic.mooo.com is 192.168.2.2 and your IPv4 address is 149.6.201.50

2022/01/05 15:00:04 INFO IPv4 address of soporte-tecnomatic.mooo.com is 192.168.2.2 and your IPv4 address is 149.6.201.50

2022/01/05 15:00:04 INFO Updating record [domain: mooo.com | host: ost-tecnomatic | provider: freedns | ip: ipv4] to use 149.6.201.50

2022/01/05 15:00:09 ERROR cannot unmarshal update response: not enough fields in response: No IP change detected for ost-tecnomatic.mooo.com with IP 149.6.201.50, skipping update

2022/01/05 15:00:09 INFO Updating record [domain: mooo.com | host: soporte-tecnomatic | provider: freedns | ip: ipv4] to use 149.6.201.50

2022/01/05 15:00:14 ERROR cannot unmarshal update response: not enough fields in response: No IP change detected for soporte-tecnomatic.mooo.com with IP 149.6.201.50, skipping update

2022/01/05 15:05:04 INFO IPv4 address of ost-tecnomatic.mooo.com is 192.168.2.2 and your IPv4 address is 149.6.201.50

2022/01/05 15:05:04 INFO IPv4 address of soporte-tecnomatic.mooo.com is 192.168.2.2 and your IPv4 address is 149.6.201.50

2022/01/05 15:05:04 INFO Updating record [domain: mooo.com | host: ost-tecnomatic | provider: freedns | ip: ipv4] to use 149.6.201.50

2022/01/05 15:05:04 ERROR cannot unmarshal update response: not enough fields in response: No IP change detected for ost-tecnomatic.mooo.com with IP 149.6.201.50, skipping update

2022/01/05 15:05:04 INFO Updating record [domain: mooo.com | host: soporte-tecnomatic | provider: freedns | ip: ipv4] to use 149.6.201.50

```

Configuration file (**remove your credentials!**):

```json


  "settings": [
    {
      "provider": "freedns",
      "domain": "mooo.com",
      "host": "ost-tecnomatic",
      "token": "token",
      "ip_version": "ipv4"
    },
    {
      "provider": "freedns",
      "domain": "mooo.com",
      "host": "soporte-tecnomatic",
      "token": "token",
     "ip_version": "ipv4"
    }
  ]
}


```

Host OS:
Debian Buster 10

**Comment by qdm12** [comment]
Sorry for the delay. It's unhealthy because the program thinks the update failed.

> cannot unmarshal update response

Although it worked. I'll fix it soon.

**Comment by qdm12** [comment]
Fixed in e4f0c0d561c0cf039b65d5f83355dff325c34559

**Comment by MilorES** [comment]
Thank you very much, I will update soon. (You could create something in the config to "ignore" errors like this... in the js)
